src/memoizer.js

```diff
@@ -42,13 +37,13 @@ function getCacheId(inputs) {
 
     var i, len, input;
 
-    for (i = 0, len = inputs.length; i < len; i += 1) {
+    for (i = 0; i < inputs.length; i++) {
```

Comment:
All major runtimes now cache length; in certain cases it can actually prevent optimizations.